src/psyclone/tests/psyir/frontend/fparser2_test.py

```diff
@@ -1829,6 +1829,38 @@ def test_handling_part_ref():
     assert new_node.name == "x"
     assert len(new_node.children) == 3  # Array dimensions
 
+    # Test a function. This will be a part_ref but will have a RoutineSymbol
```

Comment:
I think this would be better as a separate test. You could use the `fortran_reader` fixture instead of the parser then too.

Reply:
In fact, since this is such a small PR, would you mind breaking the part-ref tests out into their own file too?

Reply:
OK.